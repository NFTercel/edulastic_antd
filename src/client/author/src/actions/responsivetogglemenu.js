import { RESPONSIVE_TOGGLE_MENU } from "../constants/actions";

export const responsiveSideBar = () => ({
  type: RESPONSIVE_TOGGLE_MENU
});
