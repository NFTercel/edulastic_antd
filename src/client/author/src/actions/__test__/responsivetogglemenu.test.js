import { responsiveSideBar } from "../responsivetogglemenu";

describe("Responsive Toggle Menu", () => {
  it("responsive sidebar should return an action", () => {
    expect(responsiveSideBar()).toMatchSnapshot();
  });
});
