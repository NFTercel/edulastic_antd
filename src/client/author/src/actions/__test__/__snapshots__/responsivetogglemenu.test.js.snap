// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Responsive Toggle Menu responsive sidebar should return an action 1`] = `
Object {
  "type": "[homeUI] responsive toggle menu",
}
`;
